lib/index_group.rb

```diff
@@ -124,10 +124,12 @@ def generate_name
       "#{@name}-#{Time.now.utc.iso8601}-#{SecureRandom.uuid}".downcase
     end
 
-    def alias_map
+    def alias_map(include_closed: false)
+      expand_wildcards = ['open']
+      expand_wildcards << 'closed' if include_closed
       # Return a map of all aliases in this group, of the form:
       # { concrete_name => { "aliases" => { alias_name => {}, ... } }, ... }
-      indices = @client.indices.get_aliases
+      indices = @client.indices.get(index: "#{@name}*", expand_wildcards: expand_wildcards)
```

Comment:
Will this not return full index names instead of alias names? Or have I misunderstood how this works?

Reply:
As discussed, `get_aliases` returned the index name with aliases listed in the associated hash, `get` has the same return object.